db/system_keyspace.cc

```diff
@@ -1382,7 +1382,7 @@ schema_ptr system_keyspace::legacy::aggregates() {
 }
 
 future<> system_keyspace::setup_version(sharded<netw::messaging_service>& ms) {
-    auto& cfg = _db.local().get_config();
```

Comment:
We should get rid of db here too: pass config directly, and pass a separate interface for the schema version. Of course, need not impact this series.